authors/apps/articles/models.py

```diff
@@ -1,9 +1,11 @@
+from django.contrib.postgres import fields
 from django.db import models
-from django.utils import timezone
-from authors.apps.authentication.models import User
```

Comment:
@felixkiryowa kindly stop making unnecessary changes, for example, the User model. This may introduce bugs. Restore changes.

Reply:
I really don't understand this because I changed anything in respect to those imports 
I only added these imports
`from django.conf import settings`
`import math`

Reply:
If not they were unused imports, so there was not for them to be in that file